docs/catalyst_101.md

```diff
@@ -136,6 +136,159 @@ Once the existing members have voted to offer you membership, it’s up to you t
+
+Activity:
+
+Use the mid sprint checkin as a parallel work block to pull items into Backlog - aiming for at least 2N items in Backlog, where N is the number of people on the team. All items in Backlog should have scope and a next step:
+
+- scope: when would we consider this issue to be done? what criteria have to be met, or what behavior has to exist?
+- next step: what's the first step to getting this working?
+
+In the sprint review, instead of discussing status for each item in the sprint, prioritize the tickets that are in Backlog according to the following schema:
+
+- "Urgent" -> blocking work on high-priority tickets
+- "High" -> if we do not finish this during the next two weeks, we will not hit our quarterly goals
+- "Medium" -> we'd like to finish this during the next two weeks
+- "Low" -> if there's not other stuff to do, it's worth knocking out
+
+In the sprint planning:
+
+- Pull high priority items from in-flight and backlog into the sprint until it feels like we have enough, but not too much, work.
+  - If a low priority item is in flight, we should bump it for higher-priority work.
+
+Outside of the meetings:
+
+- If a new issue comes up, it goes in the Icebox unless there's a compelling case for it to be Urgent, in which case we talk about it as a team and pull it into the sprint if it's truly Urgent.
+
+Success criteria:
+
+- it feels easier than before to start work on a new ticket when your old ticket is done
+- minimal time spent doing homework outside of meetings
+
+Accountability/how we'll stick to it:
+
+- parallel work blocks are more engaging than "everyone talk through everything in series."
+
+#### Limit work-in-progress
+
+Goal: reduce overwhelm from "too many things going on at once" and identify bottlenecks in our process
+
+Activity:
+When moving work through the pipeline to Done:
+
+- If there are more than N+1 "things" in the next phase (N = team size) do not move your work to the next phase. Instead, find a thing in the next phase and try to move it - i.e. review someone's pull request, or see if you can help someone get their code into a reviewable state.
+  - one "thing" is: a PR, an issue, or a linked pair of issue + PR (which counts as one "thing").
+
+During mid sprint check-in and sprint review:
+
+- Thing Counting Phase at the beginning of the meeting where we count the Things and make sure we're not over the limit.
+
+Outside of meetings:
+
+- Say no to new things if they would push you over WIP limits.
+
+Success criteria:
+
+- people work together on projects more
+- people have fewer things going on at once
+
+Accountability/how we'll stick to it:
+
+- Just the in-meeting Thing Counting Phase. Maybe we can make it a race to make it engaging.
+
+### Future experiments
+
+Here are future experiments:
+
+- breaking down tasks into smaller chunks - to be more flexible in how we divvy up the work, see incremental progress, and allow for faster review turnaround time
```

Comment:
I've enjoyed using Task Lists to group smaller related tasks (or several issues) into bigger coherent chunks of work, and it sounds like this is the way GitHub is going for defining something like Epics. I like checking off the boxes and seeing the little progress bar go green. It also provides bidirectional tracks / tracked-by links on the issue pages. I think this kind of task/issue grouping would help make the process of gradually fleshing issues out as we understand more about what work is actually required, preserve some continuity of work streams across multiple sprints, give us a higher level view of what streams of work are in progress (by just looking at the issues tagged as Epics), and help keep a larger number of smaller issues from feeling cluttered or overwhelming.

Reply:
I think the projects integration with tasklists isn't *quite* there yet - or maybe it's just that we're not in the task list beta. There's a `tracks:` filter in GH projects but it doesn't let me filter to "tickets that are tracked by such-and-such epic" which would be, well, perfect.

Reply:
The public version of Task Lists doesn't add explicit metadata to the  table / board views but it does ad tracks & tracked-by links within issues, and I to me that feels good enough for organizing grouped work (and it seems like the intention when the Task Lists beta goes public is that the metadata backing the feature would then become visible in the table / board views in a standard way).